scripts/build_backend.py

```diff
@@ -85,7 +85,7 @@
             "-DDPCPP_INSTALL_DIR=" + DPCPP_ROOT,
             "-DCMAKE_C_COMPILER:PATH=" + os.path.join(DPCPP_ROOT, "bin", "clang"),
             "-DCMAKE_CXX_COMPILER:PATH=" + os.path.join(DPCPP_ROOT, "bin", "dpcpp"),
-            "-DDPCTL_ENABLE_LO_PROGRAM_CREATION=ON",
+            # "-DDPCTL_ENABLE_LO_PROGRAM_CREATION=ON",
```

Comment:
It is blocker for the PR and should be fixed before merging this PR.
LevelZero is not available.

Reply:
Ticket for fixing it https://github.com/IntelPython/dpctl/issues/361